firebase/tsconfig.functions.json

```diff
@@ -7,5 +7,6 @@
   "include": [
     "src/core",
     "src/functions"
-  ]
+  ],
+  "exclude": ["src/functions/dist/**/*"]
```

Comment:
Is this working as expected? I couldn't make it work in other projects when `exclude` is not in the main `tsconfig.json`. By "working" I mean, does VSCode suggests the import of any file undef `dist/` in the auto-complete?

Reply:
I don't think so

![Screen Shot 2021-09-27 at 17 27 07](https://user-images.githubusercontent.com/13678134/134980213-47f5da37-a21d-4cbd-ad88-ec78a29f06c0.png)